app/views/steps/_step_footer_logged_in.html.erb

```diff
@@ -0,0 +1,80 @@
+<div class="sdn-todo-list-main-action">
+  <% if current_user_step.done? %>
+    <div class="govuk-body-s sdn-todo-list-main-action__info">
+      <h2 class="govuk-heading-s">Vybavené!</h2>
+      Ak sa chcete vrátiť späť, môžete tento krok
+      návodu <%= link_to 'označiť ako nevybavený', journey_step_path(@journey, @current_step, status: 'not_started'), method: :patch, remote: true %>
+      .
+    </div>
+  <% elsif current_user_step.waiting? %>
+    <div class="govuk-body-s sdn-todo-list-main-action__info">
+      <h2 class="govuk-heading-s">Podanie bolo zaslané
```

Comment:
Myslim, si ze tu treba lepsie "copy" lebo toto nemusi vzdy znamenat, ze to bolo podanie (aj ked vacsinou ano).

Zaroven sa mi zda ten navrh prilis vela textu a vela buttonov (velkych primarnych zelenych). Toto bude treba lepsie upratat. @xpavle00 ty si trufas aj na navrh toho ako by to malo vyzerat? (Staci sa pohrat s textami a govuk elementami)

Reply:
Skusil som to zjednodusit. Prilozim screeny.